db/controllers/projects.js

```diff
@@ -1,81 +1,103 @@
-const { seq } = require('../index.js');
 const { models } = require('../index');
 
 const getProjects = async (req, res) => {
-    try {
-        const projects = await models.project.findAll();
-        res.send(JSON.stringify(projects));
-    } catch (err) {
-        res.send(`Error! ${err}`);
+    let error;
+    const projects = await models.project.findAll()
+                    .catch(err => error = err);
+    
+    if (error) {
+        res.status(400).json({ error });
```

Comment:
@raindogg I think we want a return here for these single error cases, so it's clear we aren't hitting that second res.cmd() line.

Reply:
Good call, I added that to this one and all the others in that file.